.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```
@@ -0,0 +1,143 @@
+name: 🐞 Bug Report
+description: File a bug report
+labels: ["bug", "needs-triage"]
+# assignees:
+#   - octocat
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hi there 👋,
+
+        Thank you for reporting the bug!
+
+        In case you want a new resource, data source or a new attribute supported in one of those,
+        please request it as a feature!
+  - type: input
+    id: provider-version
+    attributes:
+      label: GitLab Provider version
+      description: What version of the `gitlabhq/gitlab` provider are you using?
+      placeholder: "3.9.1"
```

Maybe I'm paranoid but I worry about laziness causing people to enter the placeholder as the value.

```suggestion
      placeholder: "X.X.X"
```